core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/attachments/Attachment.java

```diff
@@ -113,4 +121,52 @@ public String toString() {
 		return MoreObjects.toStringHelper(getClass()).add("attachmentId", attachmentId).add("fileName", fileName).toString();
 	}
 
+	/**
+	 * Downloads the file to the specified exportPath. The exportPath argument accepts both directory and file paths. Passing a file path replaces the
+	 * default file name with the desired one. If the target path is a file path and there is an existing file, it will automatically overrides the
```

Comment:
```suggestion
	 * default file name with the desired one. If the target path is a file path and there is an existing file, it will automatically overwrite the
```